Address links

nicosantangelo opened the pull request:
![image](https://user-images.githubusercontent.com/692077/34205097-6702367e-e581-11e7-91ac-d7a581273231.png)

Also adds the `ADDRESS` col name to the sidebar and capitalizes the rest too:
![image](https://user-images.githubusercontent.com/692077/34205129-86ea7870-e581-11e7-8d9c-42ac59bf251c.png)

nicosantangelo committed:
Add links to address stats on the parcel popup.

nicosantangelo committed:
Add missing addres header on sidebar

abarmat commented:
One small comment, I see that the table headers for the ongoing auctions are in CAPS. Let's set district headers in CAPS too, what do you think?

nicosantangelo commented:
Good idea! didn't realize.
Changing now

nicosantangelo committed:
Capitalize districts col names. Fix col status header

nicosantangelo commented:
Updated! and fixed status being the only header with font-weight bold on the sidebar header (from master v):

![image](https://user-images.githubusercontent.com/692077/34211735-4d5d1270-e59a-11e7-9913-f66ab233f24e.png)
